library/src/main/java/com/almeros/android/multitouch/TwoFingerGestureDetector.java

```diff
@@ -92,7 +92,7 @@ public float getCurrentSpan() {
         if (mCurrLen == -1) {
             final float cvx = mCurrFingerDiffX;
             final float cvy = mCurrFingerDiffY;
-            mCurrLen = FloatMath.sqrt(cvx*cvx + cvy*cvy);
+            mCurrLen = (float) Math.sqrt(cvx*cvx + cvy*cvy);
```

Comment:
This is needed because in android sdk ver 23 FloatMAth all method are deprecated. The implementation just copied from FloatMath.sqrt method.